src/state_machine/hashmap.rs

```diff
@@ -0,0 +1,62 @@
+#![cfg(feature="hashmap")]
```

Comment:
I think maybe this is unnecessary since it's on the module declaration in `state_machine/mod.rs`.